src/libraries/Microsoft.Extensions.Logging.Console/src/ConsoleFormatterOptions.cs

```diff
@@ -21,6 +22,7 @@ public class ConsoleFormatterOptions
         /// <summary>
         /// Gets or sets format string used to format timestamp in logging messages. Defaults to <c>null</c>.
         /// </summary>
+        [StringSyntax(StringSyntaxAttribute.DateTimeFormat)]
         public string TimestampFormat { get; set; }
```

Comment:
oh interesting.  so we're doing this for more than just arguments.  that's def something i didn't realize.  will have to update roslyn side for sure to understand that :)

Reply:
You and Jason pushed for the attribute to be for properties and fields, too. So, yeah :-)